Editor/Scripts/DCCIntegration/AutoServerCreation/AutoServerCreationSettings.cs

```diff
@@ -0,0 +1,18 @@
+﻿
+using System.Runtime.CompilerServices;
+using UnityEditor;
+using UnityEngine;
+[FilePath("Packages/com.unity.meshsync/AutoServerCreationSettings.txt", FilePathAttribute.Location.ProjectFolder)]
```

Comment:
folders/files under `Packages` folder are immutable, and they may get changed/deleted whenever Unity feels like it.
I think we should put this in `UserSettings` folder

Reply:
Changed to Assets/MeshSyncAssets. I was thinking it might be better to put it there so this can be controlled on a per project basis.